src/model.py

```diff
@@ -41,6 +40,7 @@ def __init__(self, input_dim=37, hidden_dim=100, output_dim=8, num_layers=3, **k
         self.lstm = nn.LSTM(input_dim, hidden_dim, num_layers=num_layers, batch_first=True)
         self.linear = nn.Linear(hidden_dim, output_dim)
 
+    # pylint: disable=arguments-differ
```

Comment:
Pylint messages should be kept outside the source code, to improve readability

Reply:
The purpose here was workaround the arguments differ warning from pylint. I am removing for the sake of simplicity as per your vision.